tests/k8s/1-038_validate_controller_extra_command_args/01-basic-argocd-install.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-1-38-controller-extra-command
+---
```

Comment:
I think there is no need to create a namespace. Let kuttl handle this.

Reply:
Hi Siddhesh, `redis` and `repo-server` options in the the controller StatefulSet need the namespace value. We cannot replace it with `$NAMESPACE`. Not sure if there is a way to escape and consider it as an environment variable rather than a string in the test? 

```
        - --redis
        - example-argocd-redis.test-1-38-controller-extra-command.svc.cluster.local:6379
        - --repo-server
        - example-argocd-repo-server.test-1-38-controller-extra-command.svc.cluster.local:8081
```

Reply:
Oh! Okay. Overlooked that. Let's create the namespace then.